### DISCUSSION TITLE
Installing Node.js for a site via Docker Compose file in EE4

### DISCUSSION THREAD

**Discussion by jipsa78** [EasyEngine/easyengine#1667] [Support Requests]
I have taken over development for a website, which has a custom theme that runs on Node.js.
I would like to install Node.js into the theme folder via the Docker Compose stack, as should be possible according to this help article: https://easyengine.io/handbook/customising-docker-compose-stack-in-easyengine-v4/

However, I am fairly inexperienced with both Docker and Node, so I am not sure what to put in my docker-compose-custom.yml file.
I have found some examples of Docker Compose installation of Node, but these are not EasyEngine specific and do not explain how to specify the install folder. I might be able to figure it out eventually, but if anyone has already successfully attempted this before, please share your Docker Compose code.

**Comment by gingerbeardman** [comment]
I'm trying to do this, to add `filebrowser` docker container to my site for easier uploads.

But, from the docs:

> [Limitations](https://easyengine.io/handbook/customising-docker-compose-stack-in-easyengine-v4/#limitations)
> 
> Currently, the only limitation is that you can only modify the existing services defined in default docker-compose.yml, you cannot add a new service in it, i.e. you cannot add a node service to run alongside your PHP container. This support will be added in near future.
